validation/severity.rst

```diff
@@ -155,8 +155,8 @@ so that the severity is added as an additional HTML class:
         {%- if errors|length > 0 -%}
         <ul>
             {%- for error in errors -%}
-                {% if error.cause.constraint.payload.severity is defined %}
-                    {% set severity = error.cause.constraint.payload.severity %}
+                {% if error.constraint.payload.severity is defined %}
```

Comment:
This looks like the wrong fix to me. The previous paragraph is talking about the `form_errors` block. Thus, we can assume that errors is an array of `FormErrror` instances where there is no `$constraint` property, but indeed a `$cause`.

Reply:
see #9631